KinectSaveModel/SkeletonReady.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Kinect;
+using System.IO;
+using System.Diagnostics;
+using System.Windows.Media.Media3D;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Media.Imaging;
+
+namespace KinectSaveModel
+{
+    class SkeletonReady
+    {
+        private int frameNumber = 0;
+        Skeleton[] skeleton;
+        private string path = @"C:\KinectSavedMovements\FirstMovement.txt";
+        private String ssPath = @"C:\KinectSavedMovements\screenshots\FirstMovement";
+
+        public void skeletonFrameReady(object sender, SkeletonFrameReadyEventArgs e)
+        {
+            bool dataReceived = false;
+            using (SkeletonFrame SkeletonFrame = e.OpenSkeletonFrame())
+            {
+                if (SkeletonFrame == null)
+                {
+                    Debug.WriteLine("frame == null");
+                    // Moet nog een foutmelding voor aangemaakt worden
+                }
+                else
+                {
+                    skeleton = new Skeleton[SkeletonFrame.SkeletonArrayLength];
+                    SkeletonFrame.CopySkeletonDataTo(skeleton);
+                    dataReceived = true;
+                }
+            }
+
+            if (dataReceived)
+            {
+
+                Skeleton currentSkeleton = (from s in skeleton
+                                            where s.TrackingState == SkeletonTrackingState.Tracked
+                                            select s).FirstOrDefault();
+                if (currentSkeleton != null)
```

Comment:
Dan slaat hij de coordinaten op. Dus wordt het schrijven naar het bestand niet uitgevoerd. Anders krijg je allemaal null waardes in het bestand en kan je dit niet gebruiken